eng/pipelines/templates/jobs/workload-build.yml

```diff
@@ -10,14 +10,15 @@ parameters:
 
 jobs:
 # Note: jobs.yml automatically includes the Publish Assets job for pushing the artifacts to DARC.
-- template: /eng/common/templates-official/jobs/jobs.yml
+- template: /eng/common/templates-official/jobs/jobs.yml@source
```

Comment:
Just something of note. This template (`workload-build.yml`) is explicitly used by the `eng` branch official pipeline, so `source` will always be defined. If this template would need to be used in a different pipeline, this value should be passed in as a parameter.